allensdk/brain_observatory/behavior/data_objects/eye_tracking/eye_tracking_table.py

```diff
@@ -238,7 +238,7 @@ def from_data_file(
 
 
 def get_lost_frames(
-        eye_tracking_metadata: EyeTrackingMetadataFile) -> np.ndarray:
+        eye_tracking_metadata: EyeTrackingMetadataFile) -> List[int]:
```

Comment:
How onerous would it be to leave this as a np.ndarray and change the typing on the downstream code? I think numpy arrays are more useful, as they can be used to index other numpy arrays.

If you are specifically worried about the code in `get_synchronized_frame_times`, I already fixed that with this commit on Friday afternoon:

I already fixed that with this commit

https://github.com/AllenInstitute/AllenSDK/commit/e51c39cba4af022ccb9a9ca75341ff41ffed853b

Reply:
changed per your request